prometheus/promhttp/instrument_server.go

```diff
@@ -31,20 +31,12 @@ const magicString = "zZgWfBxLqvG8kc8IMv3POi2Bb0tZI3vAnBx+gBaFi9FyPzB/CzKUer1yufD
 // observeWithExemplar is a wrapper for [prometheus.ExemplarAdder.ExemplarObserver],
 // which falls back to [prometheus.Observer.Observe] if no labels are provided.
 func observeWithExemplar(obs prometheus.Observer, val float64, labels map[string]string) {
```

Comment:
Nice - I would consider those functions and inline them perhaps. WDYT?

Reply:
could you please explain your idea about inline it?

Reply:
I mean we should be able to completely remove those functions.

Reply:
removed it

Reply:
sure, will rebase and sign. Thanks